pkg/blockdevice/new_block_device_from_file_disabled.go

```diff
@@ -1,5 +1,5 @@
-//go:build windows
-// +build windows
+//go:build !darwin && !freebsd && !linux && !windows
```

Comment:
I tend not to use 'catch-all' files like these, for the reason that if people want to build Buildbarn on other operating systems they should just figure out whether one of the existing implementations can be used. Now that we have block device support for Windows, can you please remove this file entirely? Thanks!